editor/src/components/canvas/canvas-component-entry.tsx

```diff
@@ -172,3 +178,62 @@ export class CanvasErrorBoundary extends React.Component<
     }
   }
 }
+
+interface RemoteDependencyBoundaryProps {
+  projectContents: ProjectContentTreeRoot
+  requireFn: CurriedUtopiaRequireFn | null
+}
+
+function maybeGetResolvingMessage(error: any): string | null {
+  if (error?.name === ResolvingRemoteDependencyErrorName && error?.message != null) {
+    return error.message
+  } else {
+    return null
+  }
+}
+
+interface RemoteDependencyBoundaryState {
+  resolvingMessage: string | null
+}
+export class RemoteDependencyBoundary extends React.Component<
+  RemoteDependencyBoundaryProps,
+  RemoteDependencyBoundaryState
+> {
+  constructor(props: RemoteDependencyBoundaryProps) {
+    super(props)
+    this.state = { resolvingMessage: null }
+  }
+
+  static getDerivedStateFromError(e: any): RemoteDependencyBoundaryState {
+    return {
+      resolvingMessage: maybeGetResolvingMessage(e),
+    }
+  }
+
+  componentDidUpdate(
+    prevProps: RemoteDependencyBoundaryProps,
+    _prevState: RemoteDependencyBoundaryState,
```

Comment:
You should be able to just omit the parameter right?

Reply:
...yes :upside_down_face:

Reply:
Done